src/ui/components/DismissibleTextForm/index.js

```diff
@@ -154,55 +154,48 @@ export class DismissibleTextFormBase extends React.Component<
           <div className="DismissibleTextForm-formFooter">{formFooter}</div>
         )}
         <div className="DismissibleTextForm-buttons">
-          {/*
-            These buttons each have an href so that they become anchor tags.
-            This prevents mobile taps from triggering their hover styles.
-          */}
-          {onDismiss && (
+          <span className="DismissibleTextForm-delete-submit-buttons">
```

Comment:
This diff is confusing. I just switched the order of the buttons to make them keyboard accessible. When you press tab, the submit button is now first.